strings/pangram.dart

```diff
@@ -0,0 +1,33 @@
+//https://www.geeksforgeeks.org/pangram-checking/
+//Given a string check if it is Pangram or not. A pangram is a sentence containing every letter in the English Alphabet.
+import 'package:test/test.dart';
+
+bool isPanagram(String phrase)
+{
+  List<String> alphas = 'abcdefghijklmnopqrstuvwxyz'.split(''); // all alphabets
+  var newPhrase = phrase
+      .replaceAll(' ', '')
+      .toLowerCase(); // remove spaces and take to lower case

```

Comment:
do we need to execute this? seems like some unwanted computation and memory usage. can you optimise this more?